package.json

```diff
@@ -1,5 +1,5 @@
 {
-  "name": "girl-code-it-website",
+  "name": "structuregci",
```

Comment:
@harsha14gupta Ah, name changed here?
It should be `"name": "girl-code-it-website"`